src/components/Document/List.module.css

```diff
@@ -1,14 +1,14 @@
 .unorderedList,
 .orderedList {
-	@apply pl-5 leading-7 list-none mb-6 text-f-200;
+	@apply pl-5 leading-relaxed list-none mb-6 text-f-200;
 }
 
 .unorderedList > li {
-	@apply mb-2 pl-2 list-disc;
+	@apply pl-2 list-disc;
 }
 
 .orderedList > li {
-	@apply mb-2 pl-2 list-decimal;
+	@apply pl-2 list-decimal;
 }
 
 .unorderedList > li::marker {
```

Comment:
With this change, the bullets have become cramped. See - https://docs-v2-ibw5ew6vl.thirdweb-preview.com/styleguide#unordered-list

Reply:
added a bit of bottom padding